src/layer/__tests__/BYOCLayer.ts

```diff
@@ -287,3 +287,104 @@ describe.only('shServiceRootUrl', () => {
     expect(layer.getSHServiceRootUrl()).toBe(expected);
   });
 });
+
+describe.only('shServiceRootUrl', () => {
+  beforeEach(async () => {
+    setAuthToken(AUTH_TOKEN);
+    mockNetwork.reset();
+  });
+
+  test.each([
```

Comment:
I would also include use case when  `lowResolutionCollectionId` or `lowResolutionMetersPerPixelThreshold` is not set. fftr